contracts/facets/ArchaeologistFacet.sol

```diff
@@ -163,58 +155,78 @@ contract ArchaeologistFacet {
         emit WithdrawReward(msg.sender, amountToWithdraw);
     }
 
-    /// @notice Unwraps the sarcophagus.
-    /// @dev Verifies that the unencrypted shard matches the hashedShard stored
-    /// on chain and pays the archaeologist.
+    /// @notice Publishes the raw key share for which the archaeologist is responsible during the
+    /// sarcophagus resurrection window.
+    /// Pays digging fees to the archaeologist and releases their locked bond.
+    /// Cannot be called on a compromised or buried sarcophagus.
     /// @param sarcoId The identifier of the sarcophagus to unwrap
-    /// @param unencryptedShard The archaeologist's corresponding unencrypted shard
-    function unwrapSarcophagus(bytes32 sarcoId, bytes memory unencryptedShard)
-        external
-    {
-        LibUtils.revertIfNotExistOrInactive(sarcoId);
-
-        // Confirm that the archaeologist has not already unwrapped by checking
-        // if the unencryptedShard is empty
-        LibUtils.archaeologistUnwrappedCheck(sarcoId, msg.sender);
-
-        // Confirm that the sender is an archaeologist on this sarcophagus
-        if (!LibUtils.archaeologistExistsOnSarc(sarcoId, msg.sender)) {
-            revert LibErrors.ArchaeologistNotOnSarcophagus(msg.sender);
+    /// @param rawKeyShare The keyshare the archaeologist is publishing
+    function publishKeyShare(bytes32 sarcoId, bytes calldata rawKeyShare) external {
```

Comment:
`calldata` instead of `memory` on reference type parameters saves us a copy operation when the function executes because the arguments aren't having to be copied into memory